src/card/lib/card-utils.js

```diff
@@ -314,6 +325,15 @@ export function setErrors({ isNumberValid, isCvvValid, isExpiryValid, gqlErrorsO
         }
     }
 
+    if (typeof isNameValid === 'boolean' && !isNameValid) {
```

Comment:
Can this be simplified to:

```suggestion
    if (isNameValid === false) {
```